src/views/Form/index.tsx

```diff
@@ -20,23 +20,44 @@ import { fetchApiRequest, createRequestInit } from '../../service/FetchApi';
 
 import './styles.css';
 
+type InputValidation = {
+  EMPTY: boolean,
+  INVALID: boolean,
+  OK: boolean
+}
```

Comment:
Suggestion: could we create a type that can assume one of three string values ("EMPTY", "INVALID" or "OK") and use a switch (a form of pattern matching) instead of a chain of conditionals in the `handleSubmit` function?

Reply:
The switch would be a little bit more troublesome to implement and maybe difficult code reading, it would be necessary a switch for each field and we would need to figure out a way to validate in the case both fields are invalid. Also, IMO it doesn't seem right to define a "fixed" type in this case.